packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -4256,20 +4256,46 @@ class RenderFollowerLayer extends RenderProxyBox {
 ///   * [Layer.find], for an example of how this value is retrieved.
 ///   * [AnnotatedRegionLayer], the layer this render object creates.
 class AnnotatedRegionRenderObject<T> extends RenderProxyBox {
+
+  /// Creates a new [AnnotatedRegionRenderObject] to insert [value] into the
+  /// layer tree.
+  ///
+  /// If [sized] is true, the layer is provided with the size of this render
+  /// object to clip the results of [Layer.findRegion].
+  ///
+  /// Neither [value] nor [sized] can be null.
+  AnnotatedRegionRenderObject(T value, bool sized)
+    : assert(value != null),
+      assert(sized != null),
+      _value = value,
+      _sized = sized;
```

Comment:
child?

Reply:
isn't that set by the element?

Reply:
The render object layer can be used directly, it doesn't have to be used via widgets. When used directly, the first child is usually set in the constructor. Look at the other classes in this file. :-)

Reply:
ahhh right, fixed